Explorer/Assets/DCL/UserInAppInitializationFlow/StartupOperations/SetOverrideStartingParcelOperation.cs

```diff
@@ -0,0 +1,47 @@
+using Cysharp.Threading.Tasks;
+using DCL.AsyncLoadReporting;
+using DCL.FeatureFlags;
+using ECS.SceneLifeCycle.Realm;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using Utility.Types;
+
+namespace DCL.UserInAppInitializationFlow.StartupOperations
+{
+    public class SetOverrideStartingParcelOperation : IStartupOperation
```

Comment:
I think this logic can be alltogether in `TeleportStartupOperation`. If you do that, you have the power to modify `startParcel` straight away, therefore the teleport would work just fine.
And `RealmNavigator` will know nothing of its overridden values, so you can safely remove al the extra logic added there.